pytorch_lightning/utilities/enums.py

```diff
@@ -142,6 +142,14 @@ class GradClipAlgorithmType(LightningEnum):
     VALUE = "value"
     NORM = "norm"
 
+    @staticmethod
+    def supported_type(val: str) -> bool:
```

Comment:
```suggestion
    def check_value_supported(val: str) -> bool:
```

Reply:
this is what we have in other enums.. I think it's fine.